.buildbot.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+INST_DIR=`pwd`/inst
+
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=${INST_DIR} \
+    -DLLVM_INSTALL_UTILS=On \
+    -DCMAKE_BUILD_TYPE=release \
+    -DLLVM_ENABLE_ASSERTIONS=On \
+    -DLLVM_ENABLE_PROJECTS="lld;clang" \
+    ../llvm
+make -j `nproc` install
+
+# There are many test suites for LLVM:
+# https://llvm.org/docs/TestingGuide.html
+#
+# This runs unit and integration tests.
+make -j `nproc` check-all
+cd ..
+
+# This runs the `test-suite` tests.
+# https://llvm.org/docs/TestSuiteGuide.html
+#
+# FIXME But it doesn't seem to work:
```

Comment:
I'm not sure what this comment (and commented out code) mean.

Reply:
There are three test suites and one is, confusingly, called `test-suite`. The commented code would run this suite, but it fails.

See  https://llvm.org/docs/TestingGuide.html

Reply:
Why does it fail?

Reply:
The error message is shown in the comment. I've asked on discord, but not heard back.

I've tried the master branch, the llvmorg-12.0.0 tag and the release/12.x branch :\

Reply:
OK, can we rephrase this to be clearer? Something like "The shell code below should run the `test-suite` tests but these currently fail with the following error: XXX".

Reply:
Try 92a709d8d